src/index.ts

```diff
@@ -415,6 +415,36 @@ function shouldIgnore (filename: string, ignore: RegExp[]) {
   return ignore.some(x => x.test(relname))
 }
 
+/**
+ * unsafe re-order and make .ts > .js
+ */
+function registerHandler (
+  opts: Options,
+  extensions: string[],
+  ignore: RegExp[],
+  register: Register,
+  originalJsHandler: (m: NodeModule, filename: string) => any
+) {
+  if (!opts.preferTsExts) {
+    return extensions.forEach(ext => registerExtension(ext, ignore, register, originalJsHandler))
+  }
+
+  // @todo a better way with options
+  ['.ts', '.tsx']
+    .concat(Object.keys(require.extensions), extensions) // tslint:disable-line
+    .filter((element, index, array: string[]) => array.indexOf(element) === index)
+    .forEach(ext => {
+      if (extensions.includes(ext)) {
+        registerExtension(ext, ignore, register, originalJsHandler)
+      }
+
+      const old = require.extensions[ext] // tslint:disable-line
+
```

Comment:
Remove whitespace.

Reply:
Can you confirm if you wanted this whitespace removed? I can't tell if github has pulled this comment across b/c it thinks the line hasn't changed :/